uaa/add-user.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+set -e
+
+if [ "$#" -lt 1 ]; then
+  echo
+  echo "Usage:"
+  echo "  $ ./add-user.sh <EMAIL_ADDRESS>"
+  echo
+  exit 1;
+fi
+
+if ! hash uaac 2>/dev/null; then
+  echo -n "uaac must be installed"
+  exit 1
+fi
+
+EMAIL=$1
+ORIGIN=$(awk -F '@' '{print $2}' <<< "$EMAIL")
+
+echo -n "Adding user ${EMAIL}... "
+uaac curl \
+  -X POST /Users \
+  -H "If-Match:*" \
+  -H "Accept:application/json" \
+  -H "Content-Type:application/json" \
+  -d \{\"userName\":\""${EMAIL}"\",\"emails\":[\{\"value\":\""${EMAIL}"\"\}],\"active\":true,\"verified\":true,\"origin\":\""${ORIGIN}"\"\}
```

Comment:
Quick question — why use `uaac curl` instead of one of the built-in commands, like `uaac user add`? If there's a good reason, can you add a comment documenting it here?

Reply:
Good question. Based on a test I just ran, it looks like `uaac user add` requires interactivity to specify the username, so wouldn't be helpful for automation, whereas the `uaac curl` works without the interactivity